src/drafting/bump-draft.js

```diff
@@ -0,0 +1,24 @@
+import semver from 'semver';
+import sh from 'shelljs';
+import { getLatestDigitalObject } from '../utils/get-latest.js';
+import { resolve } from 'path';
+
+export function bumpDraft(context) {
+  const { selectedDigitalObject: obj, major, patch } = context;
+  let latest = getLatestDigitalObject(context.doHome, obj.type, obj.name, obj.iri)?.version ?? '0';
+
+  const releaseType = major ? 'major' : patch ? 'patch' : 'minor';
+  latest = semver.inc(semver.coerce(latest), releaseType);
```

Comment:
Use the semver class instance to grab major, minor, patch versions for formatting. See https://github.com/npm/node-semver/blob/main/classes/semver.js#L44C13-L46

Reply:
(For the formatting, not the increment)